### PR TITLE
alsa_settings: MTLP_RVP_SDW.sh: add rt711 GE control

### PR DESCRIPTION
**Pull request by bardliao**
MTLP_RVP_SDW uese rt711 and it needs to overwritre the jack detection type.

### CODE REVIEW

**Review by kv2019i — APPROVED** [review]

**Review by marc-hb — DISMISSED** [review]
@fredoh9 ok for you?

**Comment by marc-hb** [comment]
MTL https://sof-ci.01.org/softestpr/PR1206/build523/devicetest/index.html all green but there's no MTLP_RVP_SDW model in PR testing, only in daily test runs.

CAVS25 https://sof-ci.01.org/softestpr/PR1206/build522/devicetest/index.html is totally empty LNL https://sof-ci.01.org/softestpr/PR1206/build521/devicetest/index.html had only two devices. Should not be related but worrying, will re-run.

File: alsa_settings/MTLP_RVP_SDW.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 set -e
 
+# override jack detection mode to headset
+# related linux pr: https://github.com/thesofproject/linux/pull/4969
+amixer -c sofsoundwire cset name='rt711 GE49 Selected Mode' 2 || true
```
Comment (marc-hb) [inline alsa_settings/MTLP_RVP_SDW.sh:5]: I just realized that this is a copy/paste of 90e0929368cff0 in `LNLM_SDW_AIOC.sh`

Can you please add a duplication warning at the top of the file? Something short and generic, no need to go into a detailed comparison about what is copied from `LNLM_SDW_AIOC` versus what has diverged. Something like: `# A lot of this file is duplicated in LNLM_SDW_AIOC.sh and possibly others.`

**Comment by marc-hb** [comment]
SOFCI TEST

File: alsa_settings/MTLP_RVP_SDW.sh (hunk 1 above)
Comment (bardliao) [reply alsa_settings/MTLP_RVP_SDW.sh:8]: > I just realized that this is a copy/paste of [90e0929](https://github.com/thesofproject/sof-test/commit/90e0929368cff0b29bfa92475bc4893717a5ebd1) in `LNLM_SDW_AIOC.sh`
> 
> Can you please add a duplication warning at the top of the file? Something short and generic, no need to go into a detailed comparison about what is copied from `LNLM_SDW_AIOC` versus what has diverged. Something like: `# A lot of this file is duplicated in LNLM_SDW_AIOC.sh and possibly others.`
> 
> Also add the same warning at the top of `LNLM_SDW_AIOC.sh`, which will avoid future commits like [90e0929](https://github.com/thesofproject/sof-test/commit/90e0929368cff0b29bfa92475bc4893717a5ebd1) missing the duplication and forgetting to update some files.

Done

**Review by marc-hb — APPROVED** [review]
Sorry for the delay, I missed your last force-push.

Trivial one-liner, easy to fix or revert if needed. 2 days in review, others got their chance. Let's merge this.

**Pull request merged**